front/src/modules/simulationResult/components/SpaceTimeChartWrapper/helpers/zones.ts

```diff
@@ -84,13 +90,14 @@ export function getMovableOccupancyZone(
 
   return {
     trackId,
-    trainId: formatEditoastIdToTrainScheduleId(train_schedule_id),
+    trainId,
     startTime,
     startDirection,
     endTime,
     endDirection,
-    trainName: name,
+    trainName,
     dbStartTime: startTime,
     dbEndTime: endTime,
+    isPathException: exception?.path_and_schedule !== undefined,
```

Comment:
My bad, I was wrong, if a train is an exception of path_and_schedule, it should also move when I drag an occurrence. 
But it should not move if the train is an exception of start_time.
```suggestion
    isStartTimeException: exception?.start_time !== undefined,
```